vavr/src/main/java/io/vavr/collection/CharSeq.java

```diff
@@ -1028,23 +1039,14 @@ public CharSeq takeRight(int n) {
     }
 
     @Override
-    public CharSeq takeUntil(Predicate<? super Character> predicate) {
-        Objects.requireNonNull(predicate, "predicate is null");
-        return takeWhile(predicate.negate());
+    public CharSeq takeRightUntil(Predicate<? super Character> predicate) {
+        return io.vavr.collection.Collections.takeRightUntil(this, predicate);
```

Comment:
Could you add tests for `CharSeq`? Tests in `AbstractSeqTest` do not cover this collection